imagetest/test_suites/metadata/startup_script_test.go

```diff
@@ -0,0 +1,30 @@
+// +build cit
+
+package metadata
+
+import (
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/guest-test-infra/imagetest/utils"
+)
+
+// TestStartupScript test the standard metadata script.
```

Comment:
Can you add some more context to this?

Reply:
TestStartupScript verify that the standard metadata script could run successfully by checking the output content